src/containers/VerletLists.h

```diff
@@ -262,7 +273,17 @@ class VerletLists : public LinkedCells<Particle, ParticleCell> {
     updateIdMapAoS();
     typename verlet_internal::VerletListGeneratorFunctor f(_aosNeighborLists, (this->getCutoff() * this->getCutoff()));
 
-    LinkedCells<Particle, ParticleCell>::iteratePairwiseAoS2(&f, useNewton3);
+    switch (_buildVerletListType) {
+      case BuildVerletListType::VerletAoS:
+        LinkedCells<Particle, ParticleCell>::iteratePairwiseAoS2(&f, useNewton3);
+        break;
+      case BuildVerletListType::VerletSoA:
+        LinkedCells<Particle, ParticleCell>::iteratePairwiseSoA2(&f, useNewton3);
+        break;
+      default:
+        utils::ExceptionHandler::exception("VerletLists::updateVerletListsAoS(): unsupported BuildVerletListType");
```

Comment:
add actual _buildVerletListType to the message

Reply:
fixed with 
https://github.com/AutoPas/AutoPas/pull/57/commits/93b6908b38d3a6372b47530d61a95358a3f6dd2c and
https://github.com/AutoPas/AutoPas/pull/57/commits/064f22266ba915738221a6509910591a0eaf3abc:
adds possibility to pass fmt format like strings to the exception handler and uses it here